src/nyc_trees/apps/users/models.py

```diff
@@ -15,45 +15,124 @@
 class AchievementDefinition(object):
     """A class representing an achievment definition
     name - string - title of the achievement
-    description - string - description of the achievement requirements
+    description - requirements (after achievement)
```

Comment:
I find these names confusing. Consider `description_complete` and `description_pending`.